### PR TITLE
OTC-151 : Missing statistics for uploading of health facilities into the register

### PR DESCRIPTION
**Pull request by hirensoni913**
https://openimis.atlassian.net/browse/OTC-151

### CODE REVIEW

File: IMIS/Registers.aspx.vb [hunk 1]
```diff
@@ -343,11 +343,11 @@ Partial Public Class UploadICD
                                  "<b>" & imisgen.getMessage("L_HF") & "</b><br>" &
                                  imisgen.getMessage("L_SENT") & ": " & Output("SentHF") & "<br>" &
                                  imisgen.getMessage("L_INSERTED") & ": " & Output("Inserts") & "<br>" &
-                                 imisgen.getMessage("L_UPDATED") & ": " & Output("Updates") & "</br><br>" &
-                                 "<b>" & imisgen.getMessage("L_HFCATCHMENT") & "</b><br>" &
```
Comment (delcroip) [inline IMIS/Registers.aspx.vb:347]: Why did you remove this other messages for HF ? there is still HF upload

Comment (hirensoni913) [reply IMIS/Registers.aspx.vb:347]: I removed the other part because those parameters from output parameters were removed in DAL by a previous programmer. And if I am not wrong they are also removed from SP but in the user interface they forgot to remove. And that's why it was throwing an exception and the statistics were not displaying.

Comment (hirensoni913) [reply IMIS/Registers.aspx.vb:347]: I just checked the SP, the parameters are still there. Now the question is why they were commented in DAL?

Comment (delcroip) [reply IMIS/Registers.aspx.vb:347]: Hi, it is in IMIS/Registers.aspx.vb

Comment (hirensoni913) [reply IMIS/Registers.aspx.vb:347]: yes, it is in IMIS/Register.aspx.vb

Comment (delcroip) [reply IMIS/Registers.aspx.vb:347]: For me it doesn't solve the issue, because there is no error but not statistic either

Comment (hirensoni913) [reply IMIS/Registers.aspx.vb:347]: Are you sure you have the latest Develop branch? Because in the latest branch it was not working.

Comment (hirensoni913) [reply IMIS/Registers.aspx.vb:347]: I just checked the chf dev and even there  you will find the same issue.

**Review by dragos-dobre — APPROVED** [review]
